.travis.yml

```diff
@@ -2,22 +2,76 @@
 language: c++
 sudo: required
 
+services:
+  - docker
+
 os:
   - linux
   - osx
 
-dist: trusty # Ubuntu 14.04.
+env:
+  global:
+    - BUILD_TYPE=Debug
+    - TECA_DIR=/travis_teca_dir
+    - TECA_DATA_REVISION=39
+  matrix:
+    - DOCKER_IMAGE=ubuntu IMAGE_VERSION=18.04 SCRIPT_DIR=ubuntu_18_04
+    - DOCKER_IMAGE=fedora IMAGE_VERSION=28 SCRIPT_DIR=fedora_28
```

Comment:
could we add a line(s) for Python 2 & 3 runs? see comment below about using environment var for the Python version. I know we said we'd do this in another pr, but it seems almost trivial addition...